src/options/get.rs

```diff
@@ -38,6 +40,27 @@ where
     T::get_option_value(option_name, builtin_features, opt, git_config)
 }
 
+lazy_static! {
+    static ref GIT_CONFIG_THEME_REGEX: Regex = Regex::new(r"^delta\.(.+)\.(light|dark)$").unwrap();
+}
+
+pub fn get_themes(git_config: Option<git_config::GitConfig>) -> Vec<String> {
+    let mut themes: Vec<String> = Vec::new();
+    for e in &git_config.unwrap().config.entries(None).unwrap() {
+        let entry = e.unwrap();
+        let entry_name = entry.name().unwrap();
+        let entry_value = entry.value().unwrap();
+        if entry_value == "true" {
```

Comment:
Do we actually need to look at the value here? (Also, if someone has somehow got both `light` and `dark` set in a theme would that theme be demo'd twice?)

Reply:
Good call, it's not needed. Removed.

(And just pushed a test update and a fix for the bug that would have allowed it to be demoed twice.)